---
layout: default
title: meyerlab/research
permalink: /research/
---
 
<!--<div class="post"><br>
 <h1 class="post-title">Phylogenetic Simulation</h1>
  Here is a cool coalescent simulation from <a href="https://twitter.com/trvrb">@trvrb</a>. It is a step toward implementing javascript on my site. It shows the real time generation of a phylogenetic tree, a structure that I use daily to understand influenza evolution.<br><br> 
  <div style="display:block;text-align:center"><iframe src="{{ site.baseurl }}/demos/coaltrace/" width="716" height="520"></iframe></div>
 
  <h1 class="post-title">Haplotype Simulation</h1> 
 Here is a cool haplotype simulation from <a href="https://twitter.com/trvrb">@trvrb</a>.<br><br> 
  <div style="display:block;text-align:center"><iframe src="{{ site.baseurl }}/demos/haplotypes/" width="716" height="520"></iframe></div> 
</div>-->


<div class="post"><br> 
 <h1 class="post-title">Science</h1>
 In general, I work to build predictive models of protein evolution. I am particularly interested in the way protein structure constrains and determines evolution, the evolution and epidemiology of viruses, the way evolution and epidemiology are connected, the tools and models that are used to study evolution, and how large datasets can be used to forecast the emergence and evolution of viruses. 
  <br>
  Here is a word cloud generated from the text of my first author papers. I think it does a good job outlining what I worked on during my PhD.
 <br>
 <div style="display:block;text-align:center"><img src="../img/work_wordcloud.png" width='600px'></div>
 <br>
 <h1 class="post-title">Software</h1>
 In addition to my scientific research interests, I am interested in scientific computing, open science, and computational statistics. I know and program in Python, R, Bash, and C++ as well as some intermittent Mathematica, Matlab, and Tcl/Tk. In addition, the Wilke lab maintains a public git repository of the one-off tools we've produced for various projects... <a href="https://github.com/clauswilke/WilkeLabProteinEvolutionToolbox">https://github.com/clauswilke/WilkeLabProteinEvolutionToolbox</a>.<br>
 <br>
 If there is any piece of software that seems related to what we do but is not in the repository I may have it laying around or it may be in a github repository for one of our papers.<br>
</div>

<div class="post"><br>
 <h1 class="post-title">Current Work</h1>
 <b>Geometrical constraints predict adaptive evolution in influenza hemagglutinin</b><br>
 This work is detailed in a paper that was published in <i>PLOS Pathogens</i>. I will post more details soon.<br><br>
 <b>Identifying evolutionary constraints with a biochemically meaningful protein model</b><br>
 This work appeared in <i>Science</i>. I will make a blog post on this material shortly<br>
 <br>
 <b>Time dependence of evolutionary metrics during an emerging influenza virus outbreak</b><br>
 
 For example, here is a plot of the molecular clock rate versus time. It shows the molecular clock rate of the hemagglutinin protein during the first 25 months of the 2009 pandemic flu outbreak. This will appear in a forthcoming article in <i>Virus Evolution</i>.
 <br>
 <br>
 <div style="display:block;text-align:center"><img src="../img/clock_rate.png"></div>
 <br>
 <font size='2'><b>Molecular clock rate computed by BEAST for pH1 hemagglutinin and pN1 neuraminidase from the pH1N1 outbreak.</b> In panel A, we show the molecular clock rate over time for pH1, and in panel B we show the molecular clock rate over time for pN1. The error bars represent the HPD 95% of the mean, as reported by BEAST. The plot shows a four-fold decline in the substitution rate estimates from single month of data to 25 months of aggregated data. Further, the molecular clock HPD 95% for the first 2 months of data, for both pH1 and pN1, does not overlap the final clock rate, indicating that these early estimates are in no way representative of the long-term estimates.</font>
  
  
 <!--Here is a test plot of the molecular clock rate versus time drawn on the fly by the d3 javascript library. It shows the molecular clock rate of the hemagglutinin protein during the first 25 months of the 2009 pandemic flu outbreak. This will appear in a forthcoming article in <i>Virus Evolution</i>.<br><br>
 <div style="display:block;text-align:center"><iframe src="{{ site.baseurl }}/demos/molecular_clock/" width="600" height="500"></iframe></div>-->
 
</div>

<div class="post"><br>
 <h1 class="post-title">Past Work in the Wilke Lab</h1>
 <b>Predicting viral evolution</b><br>
 By combining techniques from bioinformatics, statistical mechanics, biochemistry, molecular biology, and genetics it may be possible to predict the path of viral escape from host challenges. To accomplish this <img src="../img/force_curve.png" style="display:inline;float:left;margin:5px 10px 0px 0px" width="320">we can narrow the field of possible mutations with evolutionary intuition and bioinformatics, build a data set for training an in silico evolutionary algorithm, and test our resulting predictions experimentally.<br>
 <br>
 Our first foray into the field is to use steered molecular dynamics to pull apart the two proteins in a complex several dozens times and calculate the force that was required to successful separate the complex. <br>
 <br>
 The curve is generated with an anchored receptor and a force applied to the viral spike protein in our test system. The plot shows the average interpolated force over the replicas with p-value for difference in maximum force in the bottom left. This work combines folks from Wilke, Ellington, and Sawyer lab. This article has been published in <i>PeerJ</i> and is available <a href="http://dx.doi.org/10.7717/peerj.266">here</a>.<br>
 <br>
 <b>Biophysical constraints on protein coding sequence evolution </b><br>
 On a whole genome scale, a direct correlation between relative solvent accessibility (RSA) of sites (amino acids) in proteins and the rate of evolution (measured as rate of non-synonymous versus synonymous substitutions) at that site was established previously <a href="http://mbe.oxfordjournals.org/content/early/2009/07/13/molbev.msp146.short">here</a>, providing a reasonable model for (relatively) neutral mutations.<br>
 <br>
 Fitting the rate versus RSA curve one can find sites that differ significantly from neutrality, then, compare the predictions to the current state-of-the-art methods for functional predictions. Results from this work was published in two separate articles appearing <a href="http://dx.doi.org/10.1093/molbev/mss217">here</a> in <i>Molecular Biology and Evolution</i> and <a href="http://dx.doi.org/10.1098/rstb.2012.0334">here</a> in <i>Philosophical Transactions of the Royal Society B</i>.<br>
 <div style="display:block;text-align:center"><img src="../img/ha_plotkin_ave.png"></div>
 <font size='2'><b>Plot of dN/dS versus RSA using a model with an expected linear dependence builtin.</b>Red points represent amino acid sites identified in other studies of Hemagglutinin H1. Points that do not fall between show significant deviation from neutrality. Sites above the dashed line in the preceding plots.
 </font>
 </div>

<div class="post"><br>
 <h1 class="post-title">Past Work in the Barrick Lab</h1>

 <b>Population evolutionary simulator</b><br>
 Written in C++, <a href="http://code.google.com/p/bpopsim/">bpopsim</a>, has the ability to output several useful statistics regarding organismal evolution. It may be reasonable to compare this to something like the relative frequency of simultaneously sweeping mutations in a laboratory evolution experiment.<br>
 <br>
 Below are some preliminary results from my test runs with bpopsim. The genotype frequencies plot was assembled and plotted in R.<br>
 <div style="display:block;text-align:center"><img src="../img/muller_lod_gray_2.png"></div>
 <font size='2'><b>Evolutionary simulation of an asexual constant size population utilizing bpopsim code.</b> Blue tones show the line of descent to the final dominant genotype. Gray tones represents all lineages that died out.</font>
 <br>


<div class="post"><br>
 <h1 class="post-title">Past Work in the Sutton Lab</h1>
 My master's work, in structural biology, was completed in the lab of Dr. Bryan Sutton at Texas Tech University Health Sciences Center in Lubbock, TX. I crystallized several novel constructs of the human proteins Synaptotagmin-1 and Dysferlin; I also used fluorescence spectroscopy to measure the potential for calcium binding of each domain in the proteins.<br>
</div>
